environments/prod/prod.tfvars

```diff
@@ -595,6 +595,11 @@ frontends = [
       ]
     }
     global_exclusions = [
+      {
```

Comment:
i think you meant to do this for cmc legal? https://github.com/hmcts/azure-platform-terraform/blob/prodissuecmc/environments/prod/prod.tfvars#L666